packages/web-app-files/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export abstract class ImageDimension {
+  static readonly ThumbNail: [25, 25]
```

Comment:
```suggestion
  static readonly ThumbNail: [number, number] = [25, 25]
```
(please also refactor the name to `Thumbnail`, as that's an actual word)

Reply:
You could also choose 36x36 px for the thumbnails, since that's the dimension of the `<img>` in our file list.

Reply:
good catch, thanks